docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -344,51 +344,41 @@ Model transformation
 ====================
 
 After fitting the ``RidgeRegression`` model, the model DataFrame and cross validation DataFrame are used to apply the
-model to the block matrix DataFrame to produce predictions (*y_hat*) for each label and sample using the
-``RidgeRegression.transform`` method.
+model to the block matrix DataFrame to produce predictions (*y_hat*) for each label and sample based on the
+leave-one-chromosome-out (LOCO) scheme using the ``RidgeRegression.transform_loco`` method.
```

Comment:
nit: I think you could say that you use the fit model df to generate predictions using `transform` or `transform_loco`.